testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/harness.js

```diff
@@ -57,6 +57,19 @@ NX.TestHarness.start(
     },
 
     {
+      group: 'Search',
+      items: [
+        {
+          group: 'FT',
+          items: [
+            { url: 'testsuite-oss/search/search_basic.t.js' }
```

Comment:
In that case I wouldn't even bother adding it here, just leave it for the maven builds to pick up